### ISSUE TITLE
Horizontal Bar chart ?

### ISSUE THREAD

**Issue by nguyenhose** [reactjs/react-chartjs#152]
hi, I just check on this issue: 
https://github.com/chartjs/Chart.js/issues/73
And seem react-chartjs doesn't work with `options = { invertXY: true }`  as chartjs:
http://jsfiddle.net/xcchcaptain/ye2wdou0/4/

Maybe this option has not implemented yet. Please investigate.

**Comment by Jareechang** [comment]
please check [**Chart.js V 1.1.1**](https://github.com/chartjs/Chart.js/tree/v1.1.1/docs) docs if that option is available. I think there are plugins that you can incorporate into Chart.js to achieve this though.

Either way, you may be using a later version of Chart.js in your fiddle. 

You can always upgrade to **Chart.js V2**, there is a specific type:

``` javascript
var myBarChart = new Chart(ctx, {
    type: 'horizontalBar',
    data: data,
    options: options
});
```

**Comment by nguyenhose** [comment]
Thank you, I changed to [react-chartjs-2](https://github.com/gor181/react-chartjs-2) package, it worked like charm!

**Comment by Jareechang** [comment]
This will be available in react-chartjs 2.0 once we have a stable version.
